### PR TITLE
Sponsor banner

### PR DESCRIPTION
**Pull request by curtisadams28**
I have added the sponsor banner to core-template behind the config `show_sponsor_banner` in User > Configurations. I'm leaving things simple for now as  I copy pasted Jacks code almost verbatim but we should consider revisiting this with self-service in the future.

### CODE REVIEW

**Comment by stajs** [comment]
Need a description and changelog plz

File: kibble.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "core-template",
   "version": "0.0.1",
-  "siteUrl": "https://abccinemas.screenplus.co",
+  "siteUrl": "https://staging-test.shift72.com",
```
Comment (additiverich) [inline kibble.json:4]: Are we moving to use this site rather than abccinemas

Comment (curtisadams28) [reply kibble.json:4]: I left that in by mistake. I'll fix it now.

**Review by stajs — APPROVED** [review]
Approving, but would appreciate removing the mustache whitespace, e.g. prefer `{{end}}` over `{{ end }}`.

**Comment by curtisadams28** [comment]
This has been put on hold until the new [image model](https://dev.azure.com/S72/SHIFT72/_workitems/edit/5093) is implimented.

**Comment by curtisadams28** [comment]
I have asked around and it doesn't look like the new image model will cause any issues with this.

**Pull request merged**